storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbDatabase.java

```diff
@@ -53,7 +53,11 @@
 
   public static Database createV3(
       final RocksDbConfiguration configuration, final StateStorageMode stateStorageMode) {
-    final RocksDbInstance db = RocksDbInstanceFactory.create(configuration, V3Schema.class);
+    final RocksDbAccessor db = RocksDbInstanceFactory.create(configuration, V3Schema.class);
+    return createV3(db, stateStorageMode);
+  }
+
+  static Database createV3(final RocksDbAccessor db, final StateStorageMode stateStorageMode) {
```

Comment:
Created this `RocksDbAccessor` interface so that we could have a mock in-memory replacement for the core RocksDB access logic.  Not super happy with the current implementation, but leaving a larger refactor for the future since we have active development in this area.